private/struct2keyval.m

```diff
@@ -0,0 +1,21 @@
+function keyval = struct2keyval(s)

```

Comment:
this duplicates `ft_keyval2cfg` which you can find in fieldtrip/utilities.

Reply:
The one I wrote goes the other way; I'll rename `struct2keyval` to `ft_cfg2keyval` and put it in fieldtrip/utilities instead.